src/entities/Place/model.ts

```diff
@@ -548,10 +548,14 @@ export default class PlaceModel extends Model<PlaceAttributes> {
       )}
 
       WHERE
-        p.disabled is false AND world is true
+        p.world is true
+        ${conditional(!!options.disabled, SQL`AND p.disabled is true`)}
+        ${conditional(!options.disabled, SQL`AND p.disabled is false`)}
         ${conditional(
           options.names.length > 0,
-          SQL`AND world_name IN ${values(options.names)}`
+          SQL`AND LOWER(world_name) = ANY(SELECT LOWER(unnest(${values(
+            options.names
+          )})))`
```

Comment:
Can we check how expensive it is to perform LOWER in PG instead of using JavaScript's lowercase for the values?